modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/TerraformExecutor.java

```diff
@@ -114,9 +114,7 @@ private SystemCmdResult executeWithVariables(StringBuilder command) {
         createVariablesFile();
         command.append(" -var-file=");
         command.append(VARS_FILE_NAME);
-        SystemCmdResult systemCmdResult = execute(command.toString());
-        // cleanUpVariablesFile();
```

Comment:
i think the call to cleanupVariablesFile was commented out by mistake in the previous PRs.. can you please add this back?

Reply:
I restored it.